src/backend/impl/src/repositories/types/proposal_review_commit.rs

```diff
@@ -11,13 +11,38 @@ use super::{CommitSha, DateTime, ProposalReviewId, UserId, Uuid};
 
 pub type ProposalReviewCommitId = Uuid;
 
+#[derive(Debug, CandidType, Deserialize, Clone, PartialEq, Eq)]
+pub struct ReviewedCommitState {
+    pub matches_description: Option<bool>,
+    pub comment: Option<String>,
+    pub highlights: Vec<String>,
+}
+
+impl Display for ReviewedCommitState {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        let mut commit_details = format!(
+            "Matches description: {}",
+            self.matches_description.unwrap_or(false),
```

Comment:
Maybe it's better to put here "Unanswered" or something similar to make sure it's clear the difference between the reviewer not answering and them explicitly saying that the commit does not match it's description.

Reply:
Fixed